### PR TITLE
Pull request for code review

### PR DESCRIPTION
**Pull request by visva-dev**
This is a pull request for code review.

### CODE REVIEW

**Review by trekab — CHANGES_REQUESTED** [review]
# Changes Required!
Hi @visva-dev, impressive job so far! :+1: However, there are changes for you to implement in order to complete this project.
## Required:
- [ ] Center Apple logo in its leftmost section in the navbar just like the other links are centered.
- [ ] Position the search icon at the beginning of the search input to make appear more like the  [original](https://web.archive.org/web/20140301004610/http://www.apple.com/) site.
- [ ] The four bottom elements with images should be links. (Compare [original](https://web.archive.org/web/20140301004610/http://www.apple.com/) site.)
- [ ] Center the `footer` content to appear more like the [original](https://web.archive.org/web/20140301004610/http://www.apple.com/) site.
- [ ] The four bottom images altogether should be the same width as the top main section width. Your clones bottom images' width appears to be wider than the width of the top main image. [Screenshot](https://github.com/trekab/images/blob/master/visva-apple.png?raw=true) showing all positioning issues to fix with arrows. Try to compare the highlighted areas to the [original](https://web.archive.org/web/20140301004610/http://www.apple.com/) site.
## Optional:
- [ ] Your use of the [`<main>` tag](https://www.w3schools.com/tags/tag_main.asp) appears to be redundant with the `<body>` tag. The `<div>` with the class of 'main-container' should be the `<main>`. Please review [how to use the `<main>` tag](https://www.w3schools.com/tags/tag_main.asp) for more elaboration.

__N.B:__ In your next pull request summary description, focus on highlighting the changes your commits are introducing in the pull request.

__Kindly work on the required changes and submit another code review request!__
_Feel free to contact me on slack for any inquiries about this review_ :wink:
__Treasure Kabareebe__
[GitHub](https://github.com/trekab) | [LinkedIn](https://www.linkedin.com/in/treasure-kabareebe/) | [Twitter](https://twitter.com/TKabareebe)

**Review by trekab — CHANGES_REQUESTED** [review]
__PS:__ Make sure that `stickler` is enabled for this project!

**Comment by visva-dev** [comment]
Fixes made:

- Centered Apple logo.

- Positioned the search icon at the beginning of the search input.

- Added links to the four bottom elements with images.

- Centered the footer content.

**Comment by visva-dev** [comment]
- Aligned four bottom images altogether to the same width as the top main section width.

File: style.css [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+  border-right: 1px solid #ccc;
+  padding-right: 1rem;
+}
+
+.usa-flag a:nth-child(5) {
+  border-right: 1px solid #ccc;
+  padding-right: 1rem;
+}
+
+.usa-flag a:nth-child(6) {
```
Comment (sumancrest0001) [inline style.css]: **STATUS: REQUIRE CHANGES
Hello @visva-dev** 
You did a good job and fixed all the requested changes. But still, there is a couple of issues that need to be addressed before moving ahead.

[REQUIRED] We have to remove all the repeated code.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
[OPTIONAL] Let's add vertical line between nav elements to make it similar to the original site.

**Review by sumancrest0001 — CHANGES_REQUESTED** [review]
Kindly make the requested changes and request for another review.

Have a question about the review? DM me on slack **Suman Shrestha**
**Happy coding**

**Comment by visva-dev** [comment]
- Removed repeated code

- added a vertical line between nav elements

**Comment by addod19** [comment]
## Pull Request under review by Daniel Larbi Addo (TSE)

**Review by addod19 — CHANGES_REQUESTED** [review]
## Status: Require changes :recycle: :traffic_light: 

Hi @visva-dev, you did a great job fixing all changes recommended by the previous TSE :smile: However, I fixing those changes we left out a margin between the main content and the four-column images. Kindly find the image below for more clarification
marg

Kindy fix this comment and submit another code review :+1: 
_Happy coding_ :man_technologist: 
[Daniel Larbi Addo](https://www.linkedin.com/in/daniel-larbi-addo/)

**Comment by visva-dev** [comment]
- Reduced the height of the .main-content.

**Comment by visva-dev** [comment]
- Updated background-position of main-content div.

**Review by christianotieno — APPROVED** [review]
# PROJECT APPROVED :champagne:
Excellent job on this project. :fireworks:  Keep it up.
##
### The [clone](https://raw.githack.com/visva-dev/Solo-Apple-Project/working-branch/index.html) shows good understanding and usage of building with background and gradients. :100:
##
- The cloned page is rendering as it should and meets all the project specifications.
-  The project has a descriptive README.md and is professionally done. Good job with that.
- Pull Request has a meaningful title and a short description.
- CSS is clean. No linter errors at this point and no bad practices used as well.
- The markup as well has no errors nor warnings to show. Good job with that.
- The repo has followed all the guidelines there are to follow (GitHub flow etc.)
**All the required tasks have been fulfilled in this project. I will go ahead and mark this as complete.**
- Congratulations on the splendid job @visva-dev! :+1:
- You can now merge to master. I wish you all the very best in your next tasks 🥇 
##
Happy coding :smile:
**[Christian Otieno](https://github.com/ChristianOtieno)**
[Twitter](https://twitter.com/iamchrisotieno) [linkedin](https://www.linkedin.com/in/christianotieno/) [GitLab](https://gitlab.com/christianotieno)

**Pull request merged**